com.unity.ml-agents/CHANGELOG.md

```diff
@@ -22,6 +22,8 @@ Python training by approximately 5-10%. (#4259)
 calls to `StatsRecorder.Add()` with the same key in the same step will no
 longer overwrite each other. (#4236)
 #### ml-agents / ml-agents-envs / gym-unity (Python)
+- The supported versions of `numpy` ml-agents-envs was changed to not allow 1.19.0 or later. This was done to reflect
```

Comment:
```suggestion
- The versions of `numpy` supported by ml-agents-envs was changed to disallow 1.19.0 or later. This was done to reflect
```

Reply:
That's better, but I think it needs to be "versions ... _were_ changed". Will fix along with the merge conflicts.